src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/views/Service/overview.html.twig

```diff
@@ -4,7 +4,85 @@
 
     <div id="service-states">
     {% for service in services %}
-        <div class="service-status-container" data-service-id="{{ service.id }}"></div>
+        <div class="service-status-container">
+            <div class="service-status-title">
+                <a href="{{ path('select_service', {service: service.id }) }}">{{ service.name }}</a>
+            </div>
+            <div class="service-status-graph" data-service-id="{{ service.id }}"></div>
+            <div class="service-status-entities">
+
+                <h2>{{ 'service.overview.entitylist.title'|trans({'%name%': service.name}) }}</h2>
+
+                <table>
+                    {# Production entities #}
+                    <tr>
+                        <td colspan="4">
+                            <h3>{{ 'service.overview.entitylist.production.title'|trans }}</h3>
```

Comment:
Maybe render this in the light blue upper case style the form field labels also use. This will make the table less harsh.

Reply:
This seems to me a personal preference.